io.github.GnomeMpv.json

```diff
@@ -147,8 +147,8 @@
               "sources": [
                    {
                        "type": "archive",
-                       "url": "https://wayland.freedesktop.org/releases/wayland-protocols-1.14.tar.xz",
```

Comment:
You can remove this entirely, it was only needed for old runtime.

Reply:
Ok, should we drop ffmpeg also? I see it's available in gnome 3.30 as well.

Reply:
No because we want every codec.

Reply:
Done!